README.md

```diff
@@ -191,45 +179,37 @@ The `db` folder is one you'll be working in for a bit of time as well. This
 contains the seed file but additionally it will also contain your migrations
 which you'll be going over in the next class.
 
-In the main directory there are a couple of files your familiar with, the
-`Gemfile` and `Gemfile.lock`
+In the root directory of the application you will also see a `Gemfile` and, if you've run `bundle install`, `Gemfile.lock`
 
-### Starting a Rails Server (5 min)
+### Starting a Rails Server (5 minutes / 1:00)
 
 Let's go ahead and look at the final application before we dive into the code.
 
->if you want to follow along, clone it down and checkout the solution branch
+In the console, let's run the following commands...
 
-In my console, I'm going to run the following commands:
-
-- this loads & sets up the local dependencies
 ```bash
-$ bundle install
+$ bundle install   # Loads and sets up the local dependencies
 ```
 
-- this is similar to `psql dbcreate <database name>` (plus more)
 ```bash
-$ rake db:create
+$ rake db:create   # Creates the database, equivalent to `createdb db_name`
 ```
```

Comment:
Guys, rails 5 by convention doesn't use `rake`, it supports it but the `rails` command should be used instead of `rake`